zebra-state/src/service/queued_blocks.rs

```diff
@@ -219,18 +219,49 @@ impl QueuedBlocks {
     }
 }
 
+#[derive(Debug)]
```

Comment:
Please derive the standard set of impls on new types, so they are easier to use in future code changes.

Reply:
I started over and removed this struct, and noted.